src/Akka.Cluster.Hosting/AkkaClusterHostingExtensions.cs

```diff
@@ -237,13 +256,116 @@ public static class AkkaClusterHostingExtensions
             {
                 middle = middle.AddHocon($"akka.cluster.pub-sub = \"{role}\"");
             }
-                
+
             return middle.WithActors((system, registry) =>
             {
                 // force the initialization
                 var mediator = DistributedPubSub.Get(system).Mediator;
                 registry.TryRegister<DistributedPubSub>(mediator);
             });
         }
+
+        /// <summary>
+        /// Creates a new <see cref="ClusterSingletonManager"/> to host an actor created via <see cref="actorProps"/>.
+        ///
+        /// If <see cref="createProxyToo"/> is set to <c>true</c> then this method will also create a <see cref="ClusterSingletonProxy"/> that
```

Comment:
I think you should use `<paramref name="createProxyToo"/>` instead `<see/>`?

Reply:
Ah ok

Reply:
Done